<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialOutbound</class>
 <widget class="QWidget" name="MaterialOutbound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">;

font: 75 14pt &quot;Aparajita&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;

font: 75 14pt &quot;Aparajita&quot;;</string>
   </property>
   <property name="title">
    <string>Material Outbound</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Inquiry Model:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="GoodIDradioButton">
           <property name="text">
            <string>Goods ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="GoodsNameradioButton">
           <property name="text">
            <string>Goods Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="searchpushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(153, 149, 137); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: black; }
</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 75 10pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Prompt:Double-click to add information!</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="submitpushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton { background-color: rgb(153, 149, 137); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: black; }</string>
             </property>
             <property name="text">
              <string>submit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="backpushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton { background-color: rgb(153, 149, 137); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: black; }
</string>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
